gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-oidc/src/main/java/io/gravitee/am/gateway/handler/oauth2/service/assertion/impl/ClientAssertionServiceImpl.java

```diff
@@ -134,6 +140,10 @@
                 return Maybe.error(NOT_VALID);
             }
 
+            if (ClientAssertionServiceImpl.this.domain.usePlainFapiProfile() && !isCompliantWithFapi(jwt.getHeader().getAlgorithm().getName())) {
```

Comment:
ClientAssertionServiceImpl.this.domain.usePlainFapiProfile() -> this.domain.usePlainFapiProfile()